spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from spidermon.contrib.actions.templates import ActionWithTemplates
+from spidermon.exceptions import NotConfigured
+
+
+class SendMessage(ActionWithTemplates):
+    title = None
+    fake = False
+
+    def __init__(self,
+                 title=None,
+                 fake=None,
+                 *args, **kwargs):
+        super(SendMessage, self).__init__(*args, **kwargs)
+        self.title = title or self.title
+        self.fake = fake or self.fake
+
+        if not self.title:
+            raise NotConfigured("You must provide a title for the message.")
+
+    @classmethod
+    def from_crawler_kwargs(cls, crawler):
+        return {
+            'title': crawler.settings.get('SPIDERMON_SENTRY_MESSAGE_TITLE'),
+            'fake': crawler.settings.getbool('SPIDERMON_SENTRY_FAKE'),
+        }
+
+    def run_action(self):
+        message = self.get_message()
+        if not self.fake:
+            self.send_message(message)
+        else:
+            print('-'*40)
+            print(message)
+            print('-'*40)
+
+    def get_title(self):
+        if self.title:
+            return self.render_text_template(self.title)
+        else:
+            return 'Report Title'
+
+    def get_message(self):
+        """
+        Returns message dictionary
+        """
+        message = dict()
+
+        message['title'] = self.get_title()
+
+        if self.data.job:
+            message['job_link'] = "https://app.scrapinghub.com /p/{job_id}".format(job_id=self.data.job.key)
```

Comment:
- This line is too long (109 character).
- It also has an extra space after `.com`
- We have mixed single-quotes and double-quotes.

Reply:
I'm considering line length to 120 characters to keep it more readable. Also, i see the same length in this directory.

Reply:
120 character is fine. But there is still an extra space after `.com`

Reply:
i've already removed the extra space, please check in the diff, this view is outdated.